db/migrate/20231113235906_create_foods.rb

```diff
@@ -0,0 +1,13 @@
+class CreateFoods < ActiveRecord::Migration[7.0]
+  def change
+    create_table :foods do |t|
+      t.string :name
+      t.string :measurement_unit
+      t.decimal :price
+      t.integer :quantity
+      t.references :user, null: false, foreign_key: true
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] Hello @Gardimy great effort working on your project, but as raised by the previous reviewer I still find it difficult to migrate your files to create a copy of the database on my machine and i also get the same error:

Error message: 👇 
![food](https://github.com/Gardimy/ror_recipe_app/assets/78634841/2ae12649-4d40-4025-9b62-889524367229)

I would kindly suggest you follow the suggestion given by the previous reviewer inorder to get the issue fixed. Thank you.

Reply:
I Don't get any error when I run the migration file.

Here is the screenshot: 
![Capture](https://github.com/Gardimy/ror_recipe_app/assets/111473989/dc6db631-d826-463b-9a43-27cc3ac44fb1)